about.html

```diff
@@ -1,94 +1,140 @@
+<!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, minimum-scale=1"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet" />
     <link rel="stylesheet" href="css/style.css">
-    <script
-      src="https://kit.fontawesome.com/102965b95f.js"
-      crossorigin="anonymous"
-    ></script>
+    <script src="https://kit.fontawesome.com/102965b95f.js" crossorigin="anonymous"></script>
     <title>Document</title>
-  </head>
-  <header class="header">
-    <i class="header__menu fa fa-bars fa-2x""></i>
-  </header>
-  <body>
-    <div class="container">
-      <section class="about">
-        <div class="about__head">
-          <h1 class="about__head--name">
-            FAITH APOSTOLIC LIGHTHOUSE ANNUAL CONVENTION
-          </h1>
-          <p class="about__head--qoute">
-            Be strong, and let your heart take courage, all you who wait for the
-            LORD!
-          </p>
-          <p class="about__head--qoute verse">Psalm 31:24</p>
+</head>
+
+<body>
+    <header class="header">
+        <nav class="bar">
+            <ul class="bar__cont">
+				<li class="bar__item"> <a class="bar__item" href="#"><i class="fa fa-twitter"></i> </a></li>
+				<li class="bar__item"><a class="bar__item" href="#"><i class="fa fa-facebook"></i></a></li>
+				<li class="bar__item"><a class="bar__item" href="#">English</a></li>
+				<li class="bar__item"><a class="bar__item" href="#">My Page</a></li>
+				<li class="bar__item"><a class="bar__item" href="#">Logout</a></li>
+			</ul>
+        </nav>
+        <i class="header__menu fa fa-bars fa-2x"></i>
+        <nav class="header__nav">
+            <a href=" index.html#home" class="header__home"> <img src="images/LogoMakr-7y5JLd.png" alt="faith apostolic logo"
+                    class="header__logo"></a>
+                    <ul class="header__nav--cont">
+                        <li class="header__nav--item"><a href="#home">Home</a></li>
```

Comment:
- [ ] In **about.html**, the home button should go **index.html** page but for now it is not going. I think `index.html` work Instead of using `#home` in `href`. Since you are in about.html page, `#home` won't work.